blog/2023-01-10-pulsar-2022-year-in-review.md

```diff
@@ -0,0 +1,106 @@
+
+We are grateful to all of our community members and those in the wider open-source community who contributed to the project. They are the reasons behind every step Pulsar has made over the past year.
+
+<!--truncate-->
+
+Now, let’s take a look at some of the highlights in 2022.
+
+## New Pulsar website
+
+In May 2022, the Apache Pulsar community [launched its new website](https://lists.apache.org/thread/7nx3rm3cmpbw0ws1b62k17935xgyw2tj). The new appearance has a light blue background with the homepage structure redesigned. You can see [PIP-87](https://github.com/apache/pulsar/issues/12637) and [PIP-113](https://github.com/apache/pulsar/issues/13235) for more details. Following is a comparison of the websites.
+
+Before:
+
+![](/img/pulsar-website-before.png)
+
+After:
+
+![](/img/pulsar-new-website-after-1.png)
+
+![](/img/pulsar-new-website-after-2.png)
+
+Many community members [participated in the discussion and provided valuable feedback](https://github.com/apache/pulsar/issues/15538#issuecomment-1125602899). Currently, the Pulsar community is still working to improve the reader experience by making key information more visible and accessible.
+
+## Key events recap
+
+In 2022, the Apache Pulsar community put together a number of meetups and events across the globe to share the latest messaging and streaming technologies. Among others, two Pulsar Summits received the most attention from community members.
+
+- [Pulsar Summit San Francisco 2022](https://streamnative.io/blog/community/2022-08-25-pulsar-summit-sf-2022-community-event-recap/): This event marked the first-ever, in-person Pulsar Summit in North America with 200+ attendees. It featured 20 speakers from companies like Google, AWS, and Yahoo!, 5 keynotes on Apache Pulsar and other streaming technologies, and 12 breakout sessions on tech deep dives, use cases, and ecosystem talks. [Watch the sessions](https://streamnative.io/pulsar-summit-on-demand/).
+- [Pulsar Summit Asia 2022](pathname:///blog/2022/12/01/pulsar-summit-asia-2022-recap/): With 1,500+ registrations and 40,000+ views globally, this two-day virtual event gathered 41 speakers giving 3 keynote addresses and 36 sessions. They came from companies like ByteDance, Huawei, Tencent, Nippon Telegraph and Telephone Corporation (NTT), Yum China, Netease, vivo, Nutanix, and StreamNative. [Watch the sessions](https://www.youtube.com/playlist?list=PLqRma1oIkcWgeNb3kgzqFyg5sywH8S4yy).
+
+To stay up to date with the latest events, you can join the following meetups or groups:
+
+- [New York City Apache Pulsar Meetup](https://www.meetup.com/new-york-city-apache-pulsar-meetup/)
+- [SF Bay Area Apache Pulsar Meetup](https://www.meetup.com/sf-bay-area-apache-pulsar-meetup/)
+- [Netherlands Apache Pulsar Meetup](https://www.meetup.com/netherlands-apache-pulsar-meetup/)
+- [Japan Pulsar User Group](https://japan-pulsar-user-group.connpass.com/)
```

Comment:
May we know why not add the following groups?

• [SoCal Apache Pulsar Neighborhood Meetup Group](https://www.meetup.com/socal-apache-pulsar-meetup-group/)
• [Beijing Apache Pulsar Meetup](https://www.meetup.com/beijing-apache-pulsar-meetup-by-streamnative/)
• [NorCal Apache Pulsar Neighborhood Meetup Group](https://www.meetup.com/nor-cal-apache-pulsar-meetup-group/)

Reply:
@tuhaihe